pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -405,7 +405,7 @@ func (o *Options) Run() error {
 			if c := findClusterOperatorStatusCondition(cv.Status.Conditions, configv1.RetrievedUpdates); c != nil && c.Status == configv1.ConditionFalse {
 				fmt.Fprintf(o.ErrOut, "warning: Cannot display available updates:\n  Reason: %s\n  Message: %s\n\n", c.Reason, strings.ReplaceAll(c.Message, "\n", "\n  "))
 			} else {
-				fmt.Fprintf(o.Out, "No updates available. You may force an upgrade to a specific release image, but doing so may not be supported and may result in downtime or data loss.\n")
+				fmt.Fprintf(o.Out, "No updates available. You may still upgrade to a specific release image with --to-image, but doing so may not be supported and may result in downtime or data loss.\n")
```

Comment:
We do not need to say `but doing so may not be supported and may result in downtime or data loss` anymore for --to-imgae, do we?

Reply:
the dangerous flag is `--allow-explicit-upgrade`, which has [the `--help` paragraph][1] explaining exposure.  When `availableUpdates` is empty, the options are not updating (and waiting for new recommendations) or using `--allow-explicit-upgrade --to-image ...`.  Not updating seems fairly obvious, but I guess we could call that out in this message.  Using `--allow-explicit-upgrade --to-image ...` has risks, and the user going down that path will hear more about those risks in later commands if they go down the `oc adm upgrade` -> `oc adm upgrade --to-image ...` -> `oc adm upgrade --allow-explicit-upgrade --to-image ...` path.  But I think having some warning flags at this stage helps set the context, in case they are comfortable waiting, it avoid surprising them with warning flags after they sink time in walking more of the path towards `--allow-explicit-upgrade --to-image ...`.

[1]: https://github.com/openshift/oc/blob/3ac1b026860bbd7ccd51a8c7d1d6a14b83505b91/pkg/cli/admin/upgrade/upgrade.go#L73-L80